Add 2.7.0 section to upgrading doc (#7457)
grafana/loki: docs/sources/upgrading/_index.md

The output is incredibly verbose as it shows the entire internal config struct used to run Loki, you can play around with the diff command if you prefer to only show changes or a different style output.

## Main / Unreleased

## 2.7.0

### Loki

### Loki Canary Permission

> We introduced new `push` mode to [Loki canary](https://grafana.com/docs/loki/latest/operations/loki-canary/), that can push logs generated by Loki canary directly to given Loki URL (previously it just writes to local file and you need some agent like promtail to scrape and push it to Loki).

Reviewer: ```suggestion
The new `push` mode to [Loki canary](https://grafana.com/docs/loki/latest/operations/loki-canary/) can push logs that are generated by a Loki canary directly to a given Loki URL. Previously, it only wrote to a local file and you needed some agent, such as promtail, to scrape and push it to Loki.
```
